### ISSUE TITLE
Option to enable Secret Scanning Push Protection

### ISSUE THREAD

**Issue by jorsmatthys** [NickLiffen/ghas-enablement#66]
I believe this is a relatively new GitHub feature. It also looks relatively straightforward to add it, since it can be enabled through the endpoint used to already enable secret scanning itself. 
https://docs.github.com/en/enterprise-cloud@latest/rest/repos/repos#update-a-repository

I guess there is a related todo in the code that might be picked up as well: 
```
//TODO: I can combine this function and the function found within `enableGHAS` and pass in the `secret_scanning` or `ghas` as a var
export const enableSecretScanningAlerts = async (
```

**Comment by jorsmatthys** [comment]
Oh I see now you already made a branch named push protection :)

**Comment by NickLiffen** [comment]
@jorsmatthys would you like to give my PR a review #67 🙇 

I would love your 👀 on it 👍 feel free to propose any changes 👍

**Comment by NickLiffen** [comment]
I just had another review and merged it in 🙇 

Let me know if you have any feedback 🙇

**Comment by NickLiffen** [comment]
Fixed in #67

**Comment by jorsmatthys** [comment]
Perfect thanks :) I glanced at it earlier, but didn't have the time yet to look at it properly. Will test :)
